### PR TITLE
Issue #77, Change toast to snackbar messages

### PR DESCRIPTION
**Pull request by git-typecode**
added id for activity_hotels layout.
added Snackbar in HotelsActivity.
removed Toast in HotelsActivity.

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #(issue)
Issue #77, Change toast to snackbar messages
added id for activity_hotels layout.
added Snackbar in HotelsActivity.
removed Toast in HotelsActivity.

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [ ] `./gradlew assembleDebug assembleRelease`
- [ ] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: Android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Thu May 24 11:12:39 IST 2018
+#Tue Jul 03 11:13:27 CEST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-all.zip
```
[the remark this answers is not in the document]
Comment (git-typecode) [reply Android/gradle/wrapper/gradle-wrapper.properties]: no, but how do i remove it?

File: Android/app/src/main/res/layout/activity_hotels.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/hotellsCoordinatorLayout"
```
[the remark this answers is not in the document]
Comment (git-typecode) [reply Android/app/src/main/res/layout/activity_hotels.xml]: ok, fixed

**Comment by Swati4star** [comment]
The checks are failing. Look into those. 
Also, you may do it for other modules also, in this PR only.

**Comment by git-typecode** [comment]
I have fixed the checks also and removed the Gradle 4.6 in the project.

**Pull request closed without merging**